Remove GNU ld flag which makes it act the same as Solaris ld default.

Need to convert to portable autoconf check for linker type & send upstream.

--- gnome-calendar-3.18.1/src/Makefile.am	Thu Jun  2 14:20:27 2016
+++ gnome-calendar-3.18.1/src/Makefile.am	Thu Jun  2 14:20:35 2016
@@ -62,9 +62,6 @@
     gcal-utils.c                                          \
     gcal-utils.h
 
-gnome_calendar_LDFLAGS =                                  \
-  -Wl,--export-dynamic
-
 gnome_calendar_LDADD =                                    \
   -lm                                                     \
   $(CALENDAR_LIBS)
